src/java.xml/share/classes/jdk/xml/internal/JdkConstants.java

```diff
@@ -283,6 +283,12 @@ public final class JdkConstants {
     public static final String XML_SECURITY_PROPERTY_MANAGER =
             "jdk.xml.xmlSecurityPropertyManager";
 
+    /**
+     * System Property for the Configuration File
+     * @since 21
+     */
+    public static final String CONFIG_FILE = "java.xml.config.file";
+
```

Comment:
This isn't the name of the config file but instead is the name of the _property_ that points to the config file. At some point maybe this could be renamed, maybe to CONFIG_FILE_PROPNAME or something.